examples/cms-agilitycms/tailwind.config.js

```diff
@@ -1,39 +1,195 @@
 module.exports = {
   mode: 'jit',
-  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
+  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {
-      colors: {
-        'accent-1': '#FAFAFA',
-        'accent-2': '#EAEAEA',
-        'accent-7': '#333',
-        success: '#0070f3',
-        cyan: '#79FFE1',
+    container: {
+      screens: {
+        sm: "1200px",
+        md: "1200px",
+        lg: "1200px",
+        xl: "1200px",
+      },
+    },
+    colors: {
+      transparent: "transparent",
+
+      black: "#000",
+      white: "#fff",
+
+      agility: "#222",
+
+      primary: {
+        100: "#a273ff",
+        200: "#935bff",
+        300: "#8344ff",
+        400: "#742cff",
+        500: "#6415FF",
+        600: "#5a13e6",
+        700: "#5011cc",
+        800: "#460fb3",
+        900: "#3c0d99",
+      },
+
+      secondary: {
+        100: "#7c8ba1",
+        200: "#667892",
+        300: "#506582",
+        400: "#3a5173",
+        500: "#243E63",
+        600: "#203859",
+        700: "#1d324f",
+        800: "#192b45",
+        900: "#16253b",
+      },
+
+      gray: {
+        lightest: "#fff",
+        100: "#f7fafc",
+        200: "#edf2f7",
+        300: "#e2e8f0",
+        400: "#cbd5e0",
+        500: "#a0aec0",
+        600: "#718096",
+        700: "#4a5568",
+        800: "#2d3748",
+        900: "#1a202c",
+      },
+      red: {
+        100: "#fff5f5",
+        200: "#fed7d7",
+        300: "#feb2b2",
+        400: "#fc8181",
+        500: "#f56565",
+        600: "#e53e3e",
+        700: "#c53030",
+        800: "#9b2c2c",
+        900: "#742a2a",
+      },
+      orange: {
+        100: "#fffaf0",
+        200: "#feebc8",
+        300: "#fbd38d",
+        400: "#f6ad55",
+        500: "#ed8936",
+        600: "#dd6b20",
+        700: "#c05621",
+        800: "#9c4221",
+        900: "#7b341e",
       },
-      spacing: {
-        28: '7rem',
+      yellow: {
+        100: "#fffff0",
+        200: "#fefcbf",
+        300: "#faf089",
+        400: "#f6e05e",
+        500: "#ecc94b",
+        600: "#d69e2e",
+        700: "#b7791f",
+        800: "#975a16",
+        900: "#744210",
       },
-      letterSpacing: {
-        tighter: '-.04em',
+      green: {
+        100: "#f0fff4",
+        200: "#c6f6d5",
+        300: "#9ae6b4",
+        400: "#68d391",
+        500: "#48bb78",
+        600: "#38a169",
+        700: "#2f855a",
+        800: "#276749",
+        900: "#22543d",
       },
-      lineHeight: {
-        tight: 1.2,
+      teal: {
+        100: "#e6fffa",
```

Comment:
Should this just use the default color palette from Tailwind?

Reply:
I guess I could just extend the default tailwind color palette instead of creating a new one, thanks!